include/h_malloc.h

```diff
@@ -48,10 +48,10 @@ extern "C" {
 #endif
 
 // C standard
-void *h_malloc(size_t size);
-void *h_calloc(size_t nmemb, size_t size);
-void *h_realloc(void *ptr, size_t size);
-void *h_aligned_alloc(size_t alignment, size_t size);
+__attribute__((alloc_size(1))) void *h_malloc(size_t size);
+__attribute__((alloc_size(1, 2))) void *h_calloc(size_t nmemb, size_t size);
+__attribute__((alloc_size(2))) void *h_realloc(void *ptr, size_t size);
+__attribute__((alloc_size(2))) void *h_aligned_alloc(size_t alignment, size_t size);
```

Comment:
You might as well add `alloc_size` for this one too.

Reply:
I don't understand your comment. You want me to add `alloc_size` to `h_aligned_alloc`? It's already there :/

Reply:
I just added `alloc_align` :)